src/components/PrivateLayout/PrivateLayout.tsx

```diff
@@ -206,8 +226,7 @@ const PrivateLayout = ({ children }: any) => {
                 : (<> </>)
               }
             </SubMenu>
-
-            {(canEdit || regionAdm || regionAdmDeputy || cityAdm || cityAdmDeputy && !clubAdm && !clubAdmDeputy) ? (
+            {userAnnualReportAccess?.ViewAnnualReportsTable == true && userStatisticsAccess?.CityStatisticsFormReport == true ? (
```

Comment:
if these to condition operates with boolean variable, you could just try:
`{userAnnualReportAccess?.ViewAnnualReportsTable && userStatisticsAccess?.CityStatisticsFormReport ? (`

Reply:
Thanks, changed it